tests/test_attribute_selector_block.py

```diff
@@ -0,0 +1,51 @@
+from nio.block.terminals import DEFAULT_TERMINAL
+from nio.signal.base import Signal
+from nio.testing.block_test_case import NIOBlockTestCase
+from ..attribute_selector_block import AttributeSelector
+
+
+class TestExample(NIOBlockTestCase):
+
+    def test_blacklist_signals(self):
+        """The 'hello' signal attribute will get blacklisted"""
+        blk = AttributeSelector()
+        self.configure_block(blk, {'specify_behavior': 'BLACKLIST',
+                                   'specify_attributes': [{'item': 'hello'}]})
```

Comment:
Let's add some attributes that should be included too, just to make sure they don't all get removed.